app/bundle/src/main/scala/org/bitcoins/bundle/gui/BundleGUI.scala

```diff
@@ -106,8 +105,6 @@ object BundleGUI extends WalletGUI with BitcoinSAppJFX3 {
         datadirParser.datadir,
         serverArgParser)(system.dispatcher)
 
-    GlobalData.network = toNodeConf(appConfig).network
```

Comment:
This is the loaded from filesystem or default value. We want to track the final server value